### PR TITLE
#411: Implemented a test submitting updated employee data

### PR DESCRIPTION
**Pull request by ekondrashev**
#411 Implemented a test submitting updated comment data.
Muted a test adding a puzzle to fix it, since it fails with `Already Exists` message for the `username` and `pincode`.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#501](https://github.com/timelesslounge/timelessis/issues/501) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#501](https://github.com/timelesslounge/timelessis/issues/501) is assigned to @vryazanov[/z](https://www.0crat.com/u/vryazanov), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/5540aa96-6faa-4abd-92cf-2fa773e2ba0a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: tests/integration/it_employees_test.py [hunk 1]
```diff
@@ -94,6 +94,37 @@ def test_create(client, db_session):
...
+    employee = Employee(**data)
+    db_session.add(employee)
+    db_session.commit()
+    persisted = db_session.query(Employee).get(employee.id)
+    assert persisted.comment == "No comments"
+    data['comment'] = "One comment"
+    client.post(url_for('employee.edit', id=employee.id), data=data)
+    persisted = db_session.query(Employee).get(employee.id)
+    assert persisted.comment == "One comment"
+    """fails"""
```
Comment (vryazanov) [inline tests/integration/it_employees_test.py]: @ekondrashev I think it's more complex than it should be.
I suggest the following:
```python
from tests import factories


@pytest.mark.skip
def test_edit(client):
    employee = factories.EmployeeFactory(comment="No comments")"
    client.post(
        url_for('employee.edit', id=employee.id), data={"comment": "One comment"})
    persisted = Employee.query.get(employee.id)
    assert persisted.comment == "One comment"
    """fails"""
```
wdyt?

Comment (ekondrashev) [reply tests/integration/it_employees_test.py]: @vryazanov cool, changed!
I've left an assert chancing that persisted employee comment is `No comments` though.

**Comment by vryazanov** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 1 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vryazanov[/z](https://www.0crat.com/u/vryazanov), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#501](https://github.com/timelesslounge/timelessis/issues/501) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vryazanov According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vryazanov** [comment]
@ypshenychka  I confirm

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vryazanov[/z](https://www.0crat.com/u/vryazanov)

**Pull request merged**